src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -0,0 +1,1150 @@
+        internal const string GetSchemaTable = "GetSchemaTable";
+        internal const string GetServerTransactionLevel = "GetServerTransactionLevel";
+        internal const string Insert = "Insert";
+        internal const string Open = "Open";
+        internal const string Parameter = "Parameter";
+        internal const string ParameterBuffer = "buffer";
+        internal const string ParameterCount = "count";
+        internal const string ParameterDestinationType = "destinationType";
+        internal const string ParameterIndex = "index";
+        internal const string ParameterName = "ParameterName";
+        internal const string ParameterOffset = "offset";
+        internal const string ParameterSetPosition = "set_Position";
+        internal const string ParameterService = "Service";
+        internal const string ParameterTimeout = "Timeout";
+        internal const string ParameterUserData = "UserData";
+        internal const string Prepare = "Prepare";
+        internal const string QuoteIdentifier = "QuoteIdentifier";
+        internal const string Read = "Read";
+        internal const string ReadAsync = "ReadAsync";
+        internal const string Remove = "Remove";
+        internal const string RollbackTransaction = "RollbackTransaction";
+        internal const string SaveTransaction = "SaveTransaction";
+        internal const string SetProperties = "SetProperties";
+        internal const string SourceColumn = "SourceColumn";
+        internal const string SourceVersion = "SourceVersion";
+        internal const string SourceTable = "SourceTable";
+        internal const string UnquoteIdentifier = "UnquoteIdentifier";
+        internal const string Update = "Update";
+        internal const string UpdateCommand = "UpdateCommand";
+        internal const string UpdateRows = "UpdateRows";
+
+        internal const CompareOptions compareOptions = CompareOptions.IgnoreKanaType | CompareOptions.IgnoreWidth | CompareOptions.IgnoreCase;
+        internal const int DecimalMaxPrecision = 29;
+        internal const int DecimalMaxPrecision28 = 28;  // there are some cases in Odbc where we need that ...
+        internal const int DefaultCommandTimeout = 30;
+        internal const int DefaultConnectionTimeout = DbConnectionStringDefaults.ConnectTimeout;
+        internal const float FailoverTimeoutStep = 0.08F;    // fraction of timeout to use for fast failover connections
+
+        // security issue, don't rely upon static public readonly values - AS/URT 109635
+        static internal readonly String StrEmpty = ""; // String.Empty
+
+        internal const int CharSize = sizeof(char);
+
+
+        static internal void TimerCurrent(out long ticks)
+        {
+            ticks = DateTime.UtcNow.ToFileTimeUtc();
+        }
+
+        static internal long TimerCurrent()
+        {
+            return DateTime.UtcNow.ToFileTimeUtc();
+        }
+
+        static internal long TimerFromSeconds(int seconds)
```

Comment:
The naming of these "Timer*" methods is a bit strange.  Seems like they're really about converting to ticks, e.g. SecondsToTicks or TicksFromSeconds.